tests/Common/BranchComponentTest.php

```diff
@@ -395,6 +445,25 @@ function ($branch) use ($branchName) {
         $this->assertEquals($tokenInfo['id'], $currentVersion['creatorToken']['id']);
         $this->assertEquals($tokenInfo['description'], $currentVersion['creatorToken']['description']);
 
+        if ($this->isVersionsListImplementedForDevBranch()) {
+            // test is version created for devBranch after add new config row with custom change description
```

Comment:
Tady to není pravda, testuješ po update configuration row

Reply:
Dik upravil som to